e2e-tests/commands/insurance/declarations/complete-and-submit-declarations-forms.js

```diff
@@ -3,10 +3,9 @@
  * completes declarations forms up to the specified form to stop at
  * eg, when 'antiBribery' is passed, it will complete all forms up to and including 'antiBribery'
  * @param {String} stopSubmittingAfter: The final form to submit
- * @param {String} referenceNumber: Application reference number
  */
-const completeAndSubmitDeclarationsForms = ({ stopSubmittingAfter, referenceNumber }) => {
-  cy.completePrepareApplicationSinglePolicyType({ referenceNumber });
+const completeAndSubmitDeclarationsForms = ({ stopSubmittingAfter }) => {
+  cy.completePrepareApplicationSinglePolicyType({});
```

Comment:
```suggestion
  cy.completePrepareApplicationSinglePolicyType();
```

Wondering if it needs an empty object as an argument?

Reply:
It is required because [completePrepareApplicationSinglePolicyType](https://github.com/UK-Export-Finance/exip/blob/main/e2e-tests/commands/insurance/complete-prepare-application-section-single-policy-type.js#L45) has object structured params. Without an empty object param, the command will fail.

We have this pattern throughout.

It might be possible to declare a default empty object, but we should look into this outside of this PR

Reply:
Happy to approve this if you can please create a back log ticket for above please to be picked up as a seprate PR. Ideally we should never pass an empty object. Thanks

Reply:
Have created EMS-4123 to investigate this. Very unlikely to be picked up anytime soon. There are some instances where we will want and need to pass an empty object, but otherwise agree would be great to avoid this.